test/lib/fixtures/resolutions/yarn2/yarn.lock

```diff
@@ -0,0 +1,20 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
+
+__metadata:
+  version: 4
+
+"adm-zip@npm:0.4.8":
+  version: 0.4.8
+  resolution: "adm-zip@npm:0.4.8"
+  checksum: f08c1f94d892545327ed925ea38861cb401ff1f2b496123eb9cc0d1be57546fb0337fda3b77b75c41b8a59d165aa38d2a97a5dc5a825fc0471da3c3bb858e238
+  languageName: node
+  linkType: hard
+
+"goof@workspace:.":
+  version: 0.0.0-use.local
+  resolution: "goof@workspace:."
+  dependencies:
+    adm-zip: 0.4.7
```

Comment:
The dependency is on `0.4.7`, which does not actually appear elsewhere in the `yarn.lock`. This is expected behavior for yarn 2.